wmn-data.json

```diff
@@ -6437,8 +6437,8 @@
         "name" : "Bento",
         "uri_check" : "https://bento.me/{account}",
         "e_code" : 200,
-        "e_string" : "Create Your Bento",
-        "m_string" : "Claim",
+        "e_string" : "href=\"https://bento.me/explore\">",

```

Comment:
Please check this string. In my testing, I don't see the ending `>`.

Reply:
true